src/trackoutput.cxx

```diff
@@ -162,6 +161,10 @@ void TrackOutput::process(unsigned int nframes, Buffers* buffers)
 	}
 
 	for(unsigned int i = 0; i < nframes; i++) {
+
+		//compute master volume lag;
+		_toMasterLag += 0.05 * (_toMaster - _toMasterLag);
```

Comment:
i dont like the fact we calculate this twice, here and line 113 (for the meter). @harryhaaren any ideas?